package.json

```
@@ -4,20 +4,19 @@
   "main": "index.js",
   "private": true,
   "scripts": {
-    "build": "npm run remove-dist && npm run build:production && npm run build:static",
-    "build:production": "webpack -p --mode=production",
-    "build:test": "webpack -p --mode=production --include-tests",
-    "build:dev": "npm run remove-dist && npm run build:static && webpack --mode=development --debug --devtool cheap-source-map --output-pathinfo",
+    "build": "npm run build:production && npm run build:static",
+    "build:production": "webpack --mode=production",
+    "build:test": "webpack --mode=production --include-tests",
+    "build:dev": "webpack --mode=development --debug --devtool cheap-source-map --output-pathinfo && npm run build:static",
     "build:static": "gulp svg imagemin",
     "dev": "npm run build:dev && npm run build:static",
     "dev-zip": "npm run build:dev && gulp release",
-    "watch": "npm run remove-dist && npm run build:static && webpack --watch --mode=development --debug --devtool cheap-module-eval-source-map --output-pathinfo",
+    "watch": "npm run build:static && webpack --watch --mode=development --debug --devtool cheap-module-eval-source-map --output-pathinfo",
     "release-zip": "npm run build && gulp release",
-    "remove-dist": "rm -rf dist/*",
     "test": "webpack --mode=production --display=none && bundlesize",
     "test:visualtest": "npm run build:static && start-server-and-test storybook http-get://localhost:9001 backstopjs",
     "test:visualapprove": "backstop approve --config=tests/backstop/config.js",
-    "test:analyze": "webpack -p --mode=production --env.analyze=true --json --progress --profile > /tmp/stats.json && cp ./dist/assets/js/*.js /tmp && webpack-bundle-analyzer /tmp/stats.json",
+    "test:analyze": "webpack --mode=production --env.analyze=true --json --progress --profile > /tmp/stats.json && cp ./dist/assets/js/*.js /tmp && webpack-bundle-analyzer /tmp/stats.json",
```

Similar to above, we could DRY this up a bit with:
```suggestion
    "test:analyze": "npm run build:production -- --env.analyze=true --json --progress --profile > /tmp/stats.json && cp ./dist/assets/js/*.js /tmp && webpack-bundle-analyzer /tmp/stats.json",
```